### PR TITLE
FX focus, rotary encoder, strip-search tweaks

### PR DESCRIPTION
**Pull request by eigengrau**

### CODE REVIEW

**Comment by eigengrau** [comment]
Let me know if the FX focus thing doesn’t make sense to you, though. The 202 is the first controller of this sort that I’ve owned.

File: res/controllers/Roland_DJ-202-scripts.js [hunk 1]
```diff
@@ -166,7 +168,12 @@ DJ202.Deck = function (deckNumbers, offset) {
         } else if (this.isShifted) {
             // Strip search.
             var oldPos = engine.getValue(this.currentDeck, 'playposition');
-            var newPos = Math.max(0, oldPos + newValue / 0xff);
+            // Scale to interval [0,1].
+            newValue = newValue / 0xff;
+            // Be less sensitive in longer songs.
+            var duration = engine.getValue(this.currentDeck, 'duration');
+            newValue = newValue / duration;
```
[the remark this answers is not in the document]
Comment (eigengrau) [reply res/controllers/Roland_DJ-202-scripts.js]: Nevermind. The idea was to work around the `playposition` normalization, so turning the jog-wheel at the same rate will skip the same amount of time.

[the remark this answers is not in the document]
Comment (eigengrau) [reply res/controllers/Roland_DJ-202-scripts.js]: Amended the commit with a comment clarifying this.

**Comment by Lykos153** [comment]
I like what you did with the FX focus. For the LEDs I propose:
* LED on/off signals FX status
* LED blinks slowly: FX off and focused
* LED blinks fast: FX on and focused

>Usually I expect turning a knob left/right will translate to upwards/downwards, respectively

Me, too. I already wondered why you had changed it to start with =D

**Comment by Lykos153** [comment]
Just added the XML for the right half, so now it should be more or less complete.
(I'll be away another week or so, thus only be able to drop a comment every now and then)

**Pull request merged**